lib/tests/local_request_private_cookie-issue-368.rs

```diff
@@ -0,0 +1,31 @@
+#![feature(plugin, custom_derive, decl_macro)]
+#![plugin(rocket_codegen)]
+
+extern crate rocket;
+
+use rocket::http::{Cookie, Cookies};
+
+#[get("/")]
+fn return_private_cookie(mut cookies: Cookies) -> Option<String> {
+    match cookies.get_private("cookie_name") {
+        Some(ref cookie) => Some(String::from(cookie.value())),

```

Comment:
Do we need `ref` on the left side of this match? And, can we write `Some(cookie.value().into())` on the right side?